BUILD.gn

```diff
@@ -68,6 +68,10 @@ if (current_toolchain != "${dir_pw_toolchain}/dummy:dummy") {
         "${chip_root}/src/setup_payload",
       ]
     }
+
+    if (current_os == "android") {
+      deps += [ "${chip_root}/src/controller/java" ]
```

Comment:
```suggestion
      deps += [ "${chip_root}/src/controller/java", "${chip_root}/src/setup_payload/java" ]
```

Reply:
Thanks! I had missed we had one there too. I will try to add it and this way maybe I can even test it out .

Reply:
Added.